index.js

```diff
@@ -329,18 +316,16 @@ Fetcher.prototype.postProcessFile = function (filePath) {
 Fetcher.prototype.postProcessDownloads = function () {
   var me = this;
 
-  return new Promise(function (resolve, reject) {
-    fs.readdir(me.localPath, function (err, files) {
-      if (err) {
-        logError(err);
-        reject(err);
-        return;
-      }
-      Promise.all(files.map(function (file) {
+  return fsp.readdir(me.localPath)
+    .catch(function (err) {
```

Comment:
Yep. This specific change was using `pify()` to wrap Node.js built-ins, instead of explicitly inlining callback-to-promise conversion code everywhere. The original code called `logError()`, and the new code's `.catch()` is designed to preserve the call to `logError()`.